.github/ISSUE_TEMPLATE/golang_version_upgrade.md

```diff
@@ -1,3 +1,13 @@
+---
+name: Golang version upgrade
+about: Steps to upgrade Golang version
+title: ''
+labels: kind/enhancement
```

Comment:
I don't think we have this label in the Agones repo. It looks like the last couple of PRs we used `[kind/cleanup](https://github.com/googleforgames/agones/labels/kind%2Fcleanup)`. @markmandel do you have any preference on what label(s) to use?

Reply:
Bizarrely the k8s template also uses a label (`kind/operations`) that doesn't exist.

I would just suggest `kind/feature` here and we can fix it later.

Reply:
I'd say let's use `kind/cleanup` - since it's internal and we're incrementing Go versions, nothing more.